docs/source/eFeatures.rst

```diff
@@ -2487,3 +2517,5 @@ positive signal-amplitude value on the largest-amplitude channel.
 .. _Python efeature: https://github.com/BlueBrain/eFEL/blob/master/efel/pyfeatures/pyfeatures.py
 .. _ISI Python efeature: https://github.com/BlueBrain/eFEL/blob/master/efel/pyfeatures/isi.py
 .. _Extracellular: https://github.com/BlueBrain/eFEL/blob/master/efel/pyfeatures/extrafeats.py
+.. _bAP Python efeature: https://github.com/BlueBrain/eFEL/blob/master/efel/pyfeatures/multitrace.py
```

Comment:
This can `_Multitrace efeature` as we can have more features with multiple traces.

Reply:
Done